pages.fr/common/mongo.md

```diff
@@ -0,0 +1,20 @@
+# mongo
+
+> Client shell pour MongoDB.
+> Plus d'informations: <https://docs.mongodb.com/manual/reference/program/mongo>.
+
+- Connecte à une base de données (database) :
```

Comment:
You're using a regular space instead of a non-breaking space here (and everywhere else)
```suggestion
- Connecte à une base de données (database) :
```

Reply:
oh, alright, then. i'll use this special character :)